### ISSUE TITLE
Update JSON editor on state update

### ISSUE THREAD

**Issue by Confectus** [valentin-morice/filament-json-column#5]
I was trying to update the editor value by using the `$set` callable in Filament. By the way, only the JSON viewer is updated. 

I think the project you based it on had the same problem. If you can think of any solution that could be implemented?

https://github.com/invaders-xx/filament-jsoneditor/issues/23

**Comment by Confectus** [comment]
The only solution I find viable is to listen to some event when instantiating the JSONEditor object.

For example, it could be something like this:

let json_editor = new JSONEditor($refs.editor, options);
json_editor.set(JSON.parse(JSON.stringify(state)));

```
if(typeof json_editor !== 'undefined'){
    json_editor = new JSONEditor($refs.editor, options);
    json_editor.set(JSON.parse(JSON.stringify(state)));
} else {
    let json_editor = new JSONEditor($refs.editor, options);
    json_editor.set(JSON.parse(JSON.stringify(state)));

    window.addEventListener('json-state-updated', function(event) {
        json_editor.set(JSON.parse(JSON.stringify(event.detail.data)));
    });
}
```

**Comment by valentin-morice** [comment]
Hi @Confectus, thanks for your continuous involvement in this plugin. I'm a bit busy at the moment but I'll have a look at event listeners to allow the use of the `$set` method. If in the meantime you feel like making a pull request, I'll be happy review it.

**Comment by valentin-morice** [comment]
Solved by commit: fada4b1980037983bed79300f4e1c96ccaf19d8e

**Comment by valentin-morice** [comment]
Solved by commit: 7001ebf4c19e5bd27be75c20c72b12477388165f
